utils/now_unix.go

```diff
@@ -0,0 +1,19 @@
+//go:build unix
+
+package utils
+
+import (
+	"syscall"
+	"time"
+)
+
+// Now is a faster method to get current time
+func Now() time.Time {
+	var tv syscall.Timeval
+	if err := syscall.Gettimeofday(&tv); nil != err {
```

Comment:
```suggestion
	if err := syscall.Gettimeofday(&tv); err != nil {
```

nit. just a style preference. golang will not have `if err = nil {}` problems